### PR TITLE
Business cards

### PR DESCRIPTION
**Pull request by jilaga**

### CODE REVIEW

File: Business-card/READme.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Business card
+Font: Raleway, via google fonts
+
+Colors:
+
+- pigment green: #139D3D
+- spring bud: #A5F200
+- phthalo blue: #020887
```
Comment (bebatut) [inline Business-card/READme.md:8]: This information is the same in most new branding, so it should probably be in the main README file

**Review by bebatut — COMMENTED** [review]
Thanks.

While looking at the different Pull Request, it might be good to have a folder with business cards, flyers, and stickers in that. Maybe named `communication` or something else.
What do you think?

Can you also use only small case in the file names? And make sure that the file name is `README.md`? Thank you

File: Business-card/READme.md (hunk 1 above)
Comment (jilaga) [reply Business-card/READme.md:8]: How about prints-and-digital? It represents the content and purpose of the assets in the folder

Comment (bebatut) [reply Business-card/READme.md:8]: I am not sure to understand what you mean here

Comment (jilaga) [reply Business-card/READme.md:8]: we could have `prints-and-digital` for all the media assets; stickers, flyers, business cards, merchandise and letterheads.

Comment (bebatut) [reply Business-card/READme.md:8]: Sounds good. I was not sure I followed because you answered here to my comment on the README file 😄

Comment (jilaga) [reply Business-card/READme.md:8]: 😄sorry about that.

Comment (jilaga) [reply Business-card/READme.md:8]: going ahead to make the changes and all the corrections you suggested with the file naming and README files.

Comment (jilaga) [reply Business-card/READme.md:8]: should all the uploads to `prints-and-digital` be in one PR?

Comment (bebatut) [reply Business-card/READme.md:8]: no it can be in different PRs. But please apply the same to the other PRs. Thanks

**Pull request closed without merging**